### PR TITLE
feat: update clarity design config

### PR DESCRIPTION
**Pull request by Mavericker-1996**
Update clarity design config.

### CODE REVIEW

File: configs/alibaba-inc_clarity_design.json [hunk 1]
```diff
@@ -14,9 +14,27 @@
       ]
     },
     {
-      "url": "https://at-design.alibaba-inc.com/doc/layout-pattern/",
```
Comment (s-pace) [inline configs/alibaba-inc_clarity_design.json]: Please do not remove this URL as it is required to get a 200 and initiate the crawl

Comment (s-pace) [reply configs/alibaba-inc_clarity_design.json]: @Mavericker-1996  could you provide us a compliant .xml sitemap to make sure we do not miss any content please?

File: configs/alibaba-inc_clarity_design.json [hunk 2]
```diff
@@ -14,9 +14,27 @@
...
+      ]
+    },
+    {
+      "url": "https://at-design.alibaba-inc.com/uiless/",
+      "tags": [
+        "uiless"
+      ]
+    },
+    {
+      "url": "https://at-design.alibaba-inc.com/resource/",
```
Comment (s-pace) [inline configs/alibaba-inc_clarity_design.json]: DocSearch is only provided for documentation web pages. Please remove the start_urls item not related to documentation (namely: biz, resource, uiless)

File: configs/alibaba-inc_clarity_design.json (hunk 1 above)
Comment (s-pace) [reply configs/alibaba-inc_clarity_design.json]: You can [find more details here](https://community.algolia.com/docsearch/tips.html#use-a-sitemapxml)

File: configs/alibaba-inc_clarity_design.json (hunk 2 above)
Comment (purple-force) [reply configs/alibaba-inc_clarity_design.json]: How to define documentation?

Comment (Mavericker-1996) [reply configs/alibaba-inc_clarity_design.json]: > DocSearch is only provided for documentation web pages. Please remove the `start_urls` item not related to documentation (namely: biz, resource, uiless)

I add sitemap_urls field in my config and  update the start_urls as you say, please check them out, thx a lot ^_^

Comment (s-pace) [reply configs/alibaba-inc_clarity_design.json]: Thanks @Mavericker-1996

Documenation here means technical documentation that is helpful for the community.

As stated in [our checklist](https://community.algolia.com/docsearch/who-can-apply.html):
> Your website must be a **documentation website**. We do not index blogs or commercial content.

Please remove the `start_urls` item not related to documentation (namely: biz, resource, uiless)

Thanks

Comment (Mavericker-1996) [reply configs/alibaba-inc_clarity_design.json]: I have removed the resource in the `start_urls`，but both [biz](https://at-design.alibaba-inc.com/biz/at-text-avatar) & [uiless](https://at-design.alibaba-inc.com/uiless/uiless-extendable-nav) pages are documentation as you mentioned :)

Comment (s-pace) [reply configs/alibaba-inc_clarity_design.json]: Thanks @Mavericker-1996

The pages you have pointed me out are extremely long to render. The loading of these pages ends with a blank page. Can you let us know once these pages are available? Thanks

Comment (Mavericker-1996) [reply configs/alibaba-inc_clarity_design.json]: image
image

As you can see, we are a  React UI library website, both [biz](https://at-design.alibaba-inc.com/biz/at-text-avatar) & [uiless](https://at-design.alibaba-inc.com/uiless/uiless-extendable-nav) pages are components documentation. you can them out in the screenshots aboved. Hoping you can be available to these pages :)

Comment (s-pace) [reply configs/alibaba-inc_clarity_design.json]: Thanks for the details. I will merge it and deploy it but please be aware that due to the rendering being slow/not complete, the crawl might be partial. Please make sure you either avoid every client side rendering (JS rendering) or make sure the time need to render them is shorter than 1 second.

**Pull request merged**